openpype/plugins/publish/extract_color_transcode.py

```diff
@@ -0,0 +1,283 @@
+                        new_repre["custom_tags"] = []
+                    new_repre["custom_tags"].extend(custom_tags)
+
+                # Add additional tags from output definition to representation
+                for tag in output_def["tags"]:
+                    if not new_repre.get("tags"):
+                        new_repre["tags"] = []
+                    if tag not in new_repre["tags"]:
+                        new_repre["tags"].append(tag)
+
+                instance.data["representations"].append(new_repre)
+                added_representations = True
+
+            if added_representations:
+                self._mark_original_repre_for_deletion(repre, profile)
+
+    def _rename_in_representation(self, new_repre, files_to_convert,
+                                  output_extension):
+        """Replace old extension with new one everywhere in representation."""
+        if new_repre["name"] == new_repre["ext"]:
+            new_repre["name"] = output_extension
+        new_repre["ext"] = output_extension
+
+        renamed_files = []
+        for file_name in files_to_convert:
+            file_name, _ = os.path.splitext(file_name)
+            file_name = '{}.{}'.format(file_name,
+                                       output_extension)
+            renamed_files.append(file_name)
+        new_repre["files"] = renamed_files
+
+    def _translate_to_sequence(self, files_to_convert):
+        """Returns original list or list with filename formatted in single
+        sequence format.
+
+        Uses clique to find frame sequence, in this case it merges all frames
+        into sequence format (FRAMESTART-FRAMEEND#) and returns it.
+        If sequence not found, it returns original list
+
+        Args:
+            files_to_convert (list): list of file names
+        Returns:
+            (list) of [file.1001-1010#.exr] or [fileA.exr, fileB.exr]
+        """
+        pattern = [clique.PATTERNS["frames"]]
+        collections, remainder = clique.assemble(
+            files_to_convert, patterns=pattern,
+            assume_padded_when_ambiguous=True)
+
+        if collections:
+            if len(collections) > 1:
+                raise ValueError(
+                    "Too many collections {}".format(collections))
+
+            collection = collections[0]
+            frames = list(collection.indexes)
+            frame_str = "{}-{}#".format(frames[0], frames[-1])
+            file_name = "{}{}{}".format(collection.head, frame_str,
+                                        collection.tail)
+
+            files_to_convert = [file_name]
+
+        return files_to_convert
+
+    def _get_output_file_path(self, input_path, output_dir,
+                              output_extension):
+        """Create output file name path."""
+        file_name = os.path.basename(input_path)
+        file_name, input_extension = os.path.splitext(file_name)
+        if not output_extension:
+            output_extension = input_extension
```

Comment:
```suggestion
        file_name, input_extension = os.path.splitext(file_name)
        if not output_extension:
            output_extension = input_extension.replace(".", "")
```

If the `output_extension` is empty and we use the input extension, then we end up with double `.`.